wcc-contentful-graphql/spec/wcc/contentful/graphql/builder_spec.rb

```diff
@@ -132,18 +132,20 @@
       schema = subject.build_schema
 
       # act
-      query_string = '{
-        item: allMenuButton(filter: { field: "buttonStyle", eq: "rounded" }) {
+      query_string = '
+      query menuButtonByStyle($style: String!) {
+        buttons: allMenuButton(filter: { buttonStyle: { eq: $style } }) {
```

Comment:
Note the new style of writing the filter - more consistent with what Gatsby does.